server.js

```diff
@@ -89,16 +89,6 @@ app.get('/status', (req, res) => {
     });
 });
 
-/**
- * Robots.txt
- */
-app.get('/robots.txt', noStore, require('./controllers/robots'));
```

Comment:
We'd previously mounted these early I thin in an attempt to avoid initialising common middleware for these routes. However doing that means error handling doesn't work as expected (rendering an error page results in an error…). So moving this alongside the other utility routes like `/api`  makes sense.